CHANGES

```diff
@@ -1,4 +1,30 @@
 
+Changes with Unit 1.32.0                                         26 Feb 2024
+
+    *) Feature: conditional access logging.
+
+    *) Feature: $request_id variable contains a string that is formed using
+       random data and can be used as a unique request identifier.
+
+    *) Feature: Ruby response field arrays.
+
+    *) Bugfix: "uidmap" and "gidmap" isolation options validation.
+
+    *) Bugfix: HTTP header field value encoding could be misinterpreted in
+       Python module.
+
+    *) Bugfix: http.createServer() now accepts "options" argument introduced
+       in Node.js v9.6.0, v8.12.0.
+
+    *) Bugfix: ServerRequest.flushHeaders() implemented in Node.js module to
+       make it compatible with Next.js.
+
+    *) Bugfix: ServerRequest.httpVersion variable format in Node.js module.
+
+    *) Bugfix: correct environment variables in Go module for macOS/arm64
+       systems.
+
+
```

Comment:
OK, should be a few more things getting added to this before the release, at the least

<https://github.com/nginx/unit/pull/1000>
<https://github.com/nginx/unit/pull/1024>
<https://github.com/nginx/unit/pull/1122>

Would be nice to get this simple fix in

<https://github.com/nginx/unit/pull/999>

Reply:
Sure, will edit this one once changes pushed.

Also going to update Dockerfiles if https://github.com/nginx/unit/pull/1131 will be merged.

Reply:
Yes, that should get merged.